src/Uno.UI/UI/Xaml/Controls/TextBox/OverlayTextBoxView/OverlayTextBoxViewExtension.skia.cs

```diff
@@ -165,6 +167,51 @@ public void UpdatePosition()
 		}
 	}
 
+	public void UpdateClip()
+	{
+		var clip = CalculateClipping(_contentElement.FindFirstChild<TextBlock>(), _contentElement.GetParent() as FrameworkElement);
```

Comment:
`_contentElement.FindFirstChild` is very worrying for `TextBox`es with custom template.